.clang-format

```diff
@@ -0,0 +1,96 @@
+---
+Language:        Cpp
+# BasedOnStyle:  LLVM
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlinesLeft: false
+AlignOperands:   true
```

Comment:
Kodi code guidelines say to never align vertically

Reply:
This is for linewrapping and not for column alignment. e.g.
```

Reply:
Basically same case as `AlignAfterOpenBracket`, let's discuss it there